README.md

```diff
@@ -26,11 +30,41 @@ module "alb_wafv2" {
   name  = "alb-web-acl"
   scope = "REGIONAL"
 
-  associate_alb = true
```

Comment:
Wouldn't `associate_alb = true` still be needed for the ALB use case?

Reply:
der yes. I tried removing the variable, but re-added it back and missed this place.